code/modules/client/preference_setup/general/04_equipment.dm

```diff
@@ -40,7 +40,7 @@
 		pref.backbag = 1 //Same as above
 	character.backbag = pref.backbag
 
-	if(pref.pdachoice > 4 || pref.pdachoice < 1)
+	if(pref.pdachoice > 5 || pref.pdachoice < 1)	//ARFS edit - for ceciPDAs

```

Comment:
Same deal as above, make a copy of this into a new file and edit that instead of editing source material.